msm/pippenger.cuh

```diff
@@ -297,11 +297,12 @@ void integrate(bucket_h buckets_[], uint32_t nwins, uint32_t wbits, uint32_t nbi
 
 #ifndef SPPARK_DONT_INSTANTIATE_TEMPLATES
 template __global__
-void accumulate<bucket_t, affine_t>(bucket_t::mem_t buckets_[], uint32_t nwins,
-                                    uint32_t wbits, /*const*/ affine_t points_[],
-                                    const vec2d_t<uint32_t> digits,
-                                    const vec2d_t<uint32_t> histogram,
-                                    uint32_t sid);
+void accumulate<bucket_t, affine_t::mem_t>(bucket_t::mem_t buckets_[],
+                                           uint32_t nwins, uint32_t wbits,
+                                           /*const*/ affine_t::mem_t points_[],
```

Comment:
Does one actually have to write `affine_t::mem_t` in `<>`? I mean it's sufficient to write `bucket_t` in `<>` and then declare `buckets` as `bucket_t::mem_t`, so one would think that it would work even for `affine_t`...

Reply:
Of course there might be more nuance to it, so maybe let's not hang up on this now:-)